.github/workflows/editing-toolkit-plugin.yml

```diff
@@ -212,3 +215,43 @@ jobs:
       # This will fail if the textdomain has not been changed to "full-site-editing", which indicates an issue with the sync or build scripts.
       - name: Execute phpcs on newspack-blocks PHP files.
         run: ./vendor/bin/phpcs --standard=apps/editing-toolkit/bin/newspack-block-sync-phpcs.xml
+
+  create-release-and-attach-asset:
+    name: Create release draft and attach asset
+    needs: build
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/editing-toolkit@')
+    steps:
+      - name: Set Release Version
+        id: get_release_version
+        run: echo ::set-output name=version::$(echo $GITHUB_REF | cut -d / -f 3 | sed s/^editing-toolkit@// | sed 's/-rc./ RC/' )
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1.0.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ steps.get_release_version.outputs.version }}
```

Comment:
Is it important that the release version here matches the actual plugin version in code? Maybe we should add a check to assert that they are the same if we do need them to be the same. Or perhaps this could programmatically change the versions to match the tag?

Reply:
> Is it important that the release version here matches the actual plugin version in code? Maybe we should add a check to assert that they are the same if we do need them to be the same.

> That could work :+1: 

> Or perhaps this could programmatically change the versions to match the tag?

Not sure that's an option TBH, simply based on how git works: By definition, a tag points at a given commit. But if we then change files (to update version numbers), we have to add those changes as part of a different commit -- and we can't (or shouldn't, really) update the tag to point to _that_ commit after the fact.

Reply:
Yeah, that's true. 🤔 I think updating the version manually for every merge is going to be a pain for folks, so I'm trying to think of ways we could alleviate it.

Reply:
Obviously, I don't know if there's a hard requirement for adding the version to another commit, but it does feel like poor practice for the version to include an extra change on top of the tag.

Reply:
> Yeah, that's true. 🤔 I think updating the version manually for every merge is going to be a pain for folks, so I'm trying to think of ways we could alleviate it.

While annoying and arguably somewhat redundant, I don't consider it a hard blocker TBH. FWIW, it's a requirement for wpcomsh, and I don't think it has a huge impact there.

(In practice, it seems like menial tasks like these -- that as developers, we kind of despise and have a gut reaction to automate -- might be less of an obstacle to the developer experience than other parts :sweat_smile: )

To rephrase: I'm all for automating, but wouldn't make it a blocker. It's something we could revisit as a later optimization maybe.

Reply:
> Obviously, I don't know if there's a hard requirement for adding the version to another commit, but it does feel like poor practice for the version to include an extra change on top of the tag.

Yeah, I'd go as far as to say that that's a no-go -- the tag should unambiguously point to the correct commit that makes up the version. A number of assumptions are based on this -- among them, the auto-generation of a release for the tag.

Reply:
Right, I agree it should not be a blocker. I do think the current process pains are partly from all of these small menial tasks adding up. For example, does deleting an old ETK plugin release really take more than 15 minutes? Probably not, but having to wait 5-10 minutes on Phabricator makes it seem very difficult.